### DISCUSSION TITLE
Design crash - Using ordered list

### DISCUSSION THREAD

**Discussion by s7tya** [requarks/wiki#3717] [Error / Bug Report]
**Describe the bug**
when i write like this, the design is crash.

```markdown
1. hello
1. hi

1. say ho
```
(line 3's blank is important.)

**To Reproduce**
1. make a ordered list with a blank line
```markdown
1. hello
1. hi

1. say ho
```

**Expected behavior**
can see correct design

**Screenshots**
<img width="204" alt="ss" src="https://user-images.githubusercontent.com/53410646/73662025-76522380-46de-11ea-8048-e43a400b4c58.png">
<img width="137" alt="ss" src="https://user-images.githubusercontent.com/53410646/73662053-7eaa5e80-46de-11ea-8f0d-748e41137c38.png">

**Host Info (please complete the following information):**
 - OS: Linux - Raspbian (linux) 4.4.0-1057-aws x64
 - Wiki.js version: 2.0.12
 - Database engine: PostgreSQL 11.6

**Additional context**
sorry for my poor english

**Comment by wensonsmith** [comment]
I look at the source code , it wrapped words in a `<p>` element . suppose to be  wrapped by `<li>`
